### DISCUSSION TITLE
is there linux support?

### DISCUSSION THREAD

**Discussion by mordechai** [antonreshetov/timefall#7] [Q&A]
Would be great to be able to test this out in fedora, for those not running mac os.
If there is a way we can use the current code can you share some instructions how to do it?

**Comment by antonreshetov** [comment]
@mordechai Currently, I only release under macOS, but you can locally build for Linux.

```sh
bun install
bun run make
```

**Comment by jaumepresaspuig** [comment]
Anton, could you provide more info of how to make it run for Linux?
